docs/bin/plugin_formatter.py

```diff
@@ -132,6 +133,17 @@ def rst_xline(width, char="="):
     return char * width
 
 
+def to_boolean(value):
+    """Normalize boolean option value."""
+
+    if value.get('type') == 'bool' and 'default' in value:
+        temp = boolean(value['default'])
+        new_value = value.copy()
```

Comment:
Why this copy?

Reply:
To not mutate original dict by reference

Reply:
That's true... although in this case we're just throwing away the original so it's kinda wasteful.... it feels like we're trying to follow the wrong pattern here.....